### ISSUE TITLE
Mod Not Working

### ISSUE THREAD

**Issue by saltymofo** [zeobviouslyfakeacc/HungerRevamped#7]
Hello, I have a number of other mods you've created installed and working on v.1.94 of TLD.  I downloaded the file and moved the .dll to the Mods Folder and it isn't working with MelonLoader.  Any help would be great.  Thanks for all your hard work.

**Comment by zeobviouslyfakeacc** [comment]
Hi @saltymofo

Could it be that you forgot to install [ModSettings](https://github.com/zeobviouslyfakeacc/ModSettings/releases/download/v1.8/ModSettings.dll), which HungerRevamped depends on?

If you've already installed ModSettings, I'll probably need to see your MelonLoader log file to figure out what's wrong.
Here are instructions on how to find your log file:

> How to find your Log file:
>
> - go to your game's root folder. It's the folder that contains your `Mods` folder
> - open the `MelonLoader` folder
> - find the file called `Latest.log`
> - drag and drop that file here.
> 
> If you see `MelonLoader.ModHandler.dll` instead of `Latest.log`, Please update MelonLoader to 0.4.3

**Comment by saltymofo** [comment]
Hello and thank you for getting back to me so soon.  HungerRevamped started
working once i upgraded to MellonLoader 4.3, but now i have new problems
like my sleeping bag and other items on the ground just disappeared.  I'll
upload the Latest.log file if you don't mind taking a look at it.  there is
some red text i see upon loading the game that i've never seen before.
Thanks for your time.

**Comment by zeobviouslyfakeacc** [comment]
Sounds good! Yeah, red text in the log means that, very likely, something's broken. Fortunately, that should be easy to solve once I have the log file 😄

**Comment by saltymofo** [comment]
Did it not attach?

**Comment by zeobviouslyfakeacc** [comment]
Nope, I don't see anything here. You may have to upload files directly through the GitHub website; this likely doesn't work using email responses :)

**Comment by saltymofo** [comment]
I removed all the mods, did a clean install of melonloader 4.3 and now it's
all working aside from some crashes on exiting buildings. Thanks for the
insights and again for making this great game better
